volatility3/framework/plugins/linux/malfind.py

```diff
@@ -54,12 +54,9 @@ def _list_injections(self, task):
             vollog.debug(
                 f"Injections : processing PID {task.pid} : VMA {vma_name} : {hex(vma.vm_start)}-{hex(vma.vm_end)}"
             )
-            if (
-                vma.is_suspicious(proc_layer)
-                and vma.get_name(self.context, task) != "[vdso]"
-            ):
+            if vma.is_suspicious(proc_layer) and vma_name != "[vdso]":
                 data = proc_layer.read(vma.vm_start, 64, pad=True)
-                yield vma, data
+                yield vma, vma_name, data
```

Comment:
It's an internal/non-exposed function, so this change is fine, but could we get a return type in the function signature please?

Reply:
Added